app/helpers/google_analytics_helper.rb

```diff
@@ -1,5 +1,14 @@
 module GoogleAnalyticsHelper
 
+  def google_analytics_tracking_id
+    # Use a separate GA Property for tracking embedded content
+    if params[:action] == 'embed'
+      id = "UA-60894743-2"
```

Comment:
Good idea. I think I should make a GOOGLE_ANALYTICS_EMBED_ID: UA-60894743-2 then just use that if present.